shell/src/main/java/alluxio/cli/fs/command/DistributedLoadUtils.java

```diff
@@ -62,22 +66,26 @@ public static void distributedLoad(AbstractDistributedJobCommand command,
    * @param filePath The {@link AlluxioURI} path to load into Alluxio memory
    * @param replication Number of block replicas of each loaded file
    * @param workerSet A set of worker hosts to load data
+   * @param excludeWorkerSet A set of worker hosts can not to load data
```

Comment:
Same here

Reply:
Thanks for suggest, I have modified the code.